docs/resources/Channel.md

```diff
@@ -361,6 +361,15 @@ Represents a message sent in a channel within Discord.
 }
 ```
 
+### Followed Channel Object
+
+###### Followed Channel Structure
+
+| Field | Type                                                       | Description                                       |
+| ----- | ---------------------------------------------------------- | ------------------------------------------------- |
+| channel_id | string                             | source channel id
```

Comment:
it would be more consistent to type snowflake ids as `snowflake` rather than `string`